module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraServicePrincipalDelegatedPermissionClassification.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Remove-EntraServicePrincipalDelegatedPermissionClassification.
```

Comment:
Could you help me understand the reason we are converting the contents of this file to `Add/addition`  aspects yet the file is a `Remove` action? The whole file seems to have mixed context.